.yarnrc.yml

```
@@ -1,5 +1,7 @@
+approvedGitRepositories:
+  - "**"
+enableScripts: false
 nodeLinker: node-modules
-
-npmPublishAccess: public
-
-yarnPath: .yarn/releases/yarn-4.6.0.cjs
+npmMinimalAgeGate: 7d
+yarnPath: .yarn/releases/yarn-4.14.1.cjs
+npmPreapprovedPackages: [ "@visitscotland/component-library" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)